api/src/main/resources/jakarta/servlet/resources/web-app_2_4.xsd

```diff
@@ -614,6 +614,7 @@
     <xsd:simpleContent>
       <xsd:restriction base="j2ee:string">
 	<xsd:enumeration value="GET"/>
+	<xsd:enumeration value="PATCH"/>
```

Comment:
We can't just change a released XSD, we need a new version.

Reply:
Removed.